include/fastrtps/publisher/PublisherListener.h

```diff
@@ -37,12 +38,32 @@ class RTPS_DllAPI PublisherListener
 public:
 	PublisherListener(){};
 	virtual ~PublisherListener(){};
+
 	/**
 	 * This method is called when the Publisher is matched (or unmatched) against an endpoint.
 	 * @param pub Pointer to the associated Publisher
 	 * @param info Information regarding the matched subscriber
 	 */
-	virtual void onPublicationMatched(Publisher* pub, rtps::MatchingInfo& info){(void)pub; (void)info;};
+    virtual void onPublicationMatched(
```

Comment:
Indent all the file. There are different indentations along the file.

Reply:
Done